config-wdio/wdio.local.conf.ts

```diff
@@ -206,6 +218,21 @@ export const config: WebdriverIO.Config = {
         disableWebdriverScreenshotsReporting: true,
       },
     ],
+
+    [
+      reportportal,
+      {
+        reportPortalClientConfig: {
+          token: "39dba234-6d5e-4582-bc65-0b4eb5eeeb32",
+          endpoint: "http://54.219.33.119:4000/api/v1",
+          launch: "Groot_test_execution",
+          project: "hello",
+          mode: "DEFAULT",
+          debug: false,
+          description: "Groot with Analytics",
+        },
```

Comment:
rpClient object is created in #25 line with the same piece of code, can we use the same object here instead of re-writing all the RP configs.

Reply:
Hi @balualtimetrik , as shown to you, when we pass the object of rpClient, it does not report it to TARA. Hence keeping as it is.